heft-plugins/heft-serverless-stack-plugin/package.json

```diff
@@ -11,15 +11,17 @@
   "types": "dist/heft-serverless-stack-plugin.d.ts",
   "license": "MIT",
   "scripts": {
-    "build": "heft build --clean",
     "start": "heft test --clean --watch",
-    "_phase:build": "heft build --clean",
-    "_phase:test": "heft test --no-build"
+    "build": "heft build --clean",
+    "_phase:build": "heft run --only build -- --clean",
+    "_phase:test": "heft run --only test -- --clean"
   },
   "peerDependencies": {
     "@rushstack/heft": "^0.45.14"
   },
   "dependencies": {
+    "@rushstack/heft-webpack4-plugin": "workspace:*",
+    "@rushstack/heft-webpack5-plugin": "workspace:*",
```

Comment:
Similar to `@rushstack/heft-dev-cert-plugin`, this plugin shouldn't have a dependency on these.